app/javascript/components/pages/Favorites.js

```diff
@@ -5,7 +5,34 @@ class Favorites extends Component {
     return (
       <>
         <div className="body-container">
+          <>
```

Comment:
do you need react fragments inside this div? Also, run a formatter on this document.